services/QuillLMS/app/assets/stylesheets/pages/premium_hub/premium-hub.scss

```diff
@@ -502,6 +502,58 @@
         background-color: rgba(255, 255, 255, 0.50);
         text-decoration: none;
       }
+      &.premium-reports-tab {
+        position: relative;
+        button {
+          background: transparent;
+          span::after {
+            margin-left: 8px;
+            content: "\f077";
+            font-family: "Font Awesome 5 Pro";
+          }
+          &.open {
+            span::after {
+              content: "\f078";
+            }
+          }
+          &.open:not(.active) {
+            background-color: rgba(255, 255, 255, 0.5);
+            text-decoration: none;
+            color: #FFF;
+          }
+        }
+        .premium-reports-nav-dropdown {
+          position: absolute;
+          border-radius: 8px;
+          border: 1px solid #CFCFCF;
+          background: #FFF;
```

Comment:
I think we want this and the color value above to be `$quill-white`

Reply:
good spot!